libr/asm/p/asm_x86_cs.c

```diff
@@ -2,7 +2,7 @@
 
 #include <r_asm.h>
 #include <r_lib.h>
-#include <capstone/capstone.h>
+#include <capstone.h>
```

Comment:
I was checking capstone pkg-config and it seems that it is including the capstone directory directly.. but i didnt tested on real world installations. Did you?

Reply:
See https://github.com/radareorg/radare2/pull/17563#issuecomment-685442696